docs/core-concepts.md

```diff
@@ -75,16 +75,35 @@ const context = await browser.newContext({
 
 <br/>
 
-## Pages and frames
+## Pages, frames and navigation
 
 A Browser context can have multiple pages. A [`Page`](../api.md#class-page)
-refers to a single tab or a popup window within a browser context. A page can be used to navigate
-to URLs and then interact with elements.
+refers to a single tab or a popup window within a browser context.
 
 ```js
+// Create a page.
 const page = await context.newPage();
+```
+
+A page can be used to navigate to URLs and then interact with contents. You don't usually need to worry about navigation and can focus on the interaction.
```

Comment:
Drop "You don't usually need to worry about navigation and can focus on the interaction." If I don't need to worry, as you said.